samples/js/jasmine/src/test/kotlin/org/atriumlib/samples/jasmine/SampleJsTest.kt

```diff
@@ -27,6 +27,7 @@ class SampleJsTest {
             throw IllegalArgumentException("oho... hello btw")
         }.toThrow<IllegalArgumentException> {
             messageContains("hello")
+            messageContains("jasmine")
```

Comment:
we want to check that it does not contain jasmine, your task is to filter it out. use:
```
message {
  contains("hello")
  //TODO your code
}
```

Reply:
you wan't to change messageContains method right? like this:
`    @Test
    fun expectAnExceptionWithAMessageOccurred() {
        expect {
            throw IllegalArgumentException("oho... hello btw")
        }.toThrow<IllegalArgumentException> {
            message {
                contains("hello")
                contains("jasmine")
            }
        }
    }`

Reply:
No. The test currently fails because you are asserting the wrong thing. You filtered jasmine out of the stacktrace. Why would you expect it to be there?

Reply:
I can give you the answer if you want but I thought you might want to come up with it yourself. see hint above.
instead of 
```
            messageContains("hello")
            messageContains("jasmine")
```
you have to use 
```
message {
  contains("hello")
  //TODO your code
}
```